pkg/kubelet/config/file_linux.go

```diff
@@ -40,7 +41,15 @@ const (
 	podDelete
 )
 
-func (s *sourceFile) watch() error {
+func (s *sourceFileListerWatcher) watch() {
+	go wait.Forever(func() {
+		if err := s.doWatch(); err != nil {
+			glog.Errorf("unable to read config path %q: %v", s.path, err)
+		}
+	}, s.period)
```

Comment:
Yeah, I just meant it could be a tight loop, although if it was a non-retryable error that would be a problem. Is there a similar utility method that does an exponential backoff?